<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="windowLabel">
        <property name="styleSheet">
         <string notr="true">font-size: 20px</string>
        </property>
        <property name="text">
         <string>Создание дела</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Название:</string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleInput"/>
      </item>
      <item>
       <widget class="QLabel" name="categoryLabel">
        <property name="text">
         <string>Категория:</string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categoryInput"/>
      </item>
      <item>
       <widget class="QLabel" name="repeatLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Повтор:</string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="MondayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 5px solid #0f0;</string>
          </property>
          <property name="text">
           <string>ПН</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="TuesdayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 5px solid #0f0;</string>
          </property>
          <property name="text">
           <string>ВТ</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="WednesDay">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 5px solid #0f0;</string>
          </property>
          <property name="text">
           <string>СР</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ThursdayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 5px solid #0f0;</string>
          </property>
          <property name="text">
           <string>ЧТ</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="FridayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 5px solid #0f0;</string>
          </property>
          <property name="text">
           <string>ПТ</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SaturdayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 5px solid #0f0;</string>
          </property>
          <property name="text">
           <string>СБ</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SundayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 5px solid #0f0;</string>
          </property>
          <property name="text">
           <string>ВС</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="deadlineFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="deadlineLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QCheckBox" name="deadlineCheckBox">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Дедлайн</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="deadlineInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Описание:</string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="descriptionInput"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="weekButtons"/>
 </buttongroups>
</ui>
